### PR TITLE
USWDS-Site - Changelog: Tooltip hoverable fix [#5919] 

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Add changelog entry for tooltip hoverable fix

## Related PR

https://github.com/uswds/uswds/pull/5919

## Preview link
[Tooltip  changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-5919/components/tooltip/#latest-updates)

### CODE REVIEW

File: _data/changelogs/component-tooltip.yml [hunk 1]
```diff
@@ -5,6 +5,14 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Updated component behavior to allow users to hover over tooltip content.
+    summaryAdditional: This allows the component to meet the "hoverable" standard outlined in [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).
```
Comment (mahoneycm) [inline _data/changelogs/component-tooltip.yml:10]: While I generally prefer the display of content in this link, I feel like we should update this link to match the one used in uswds/uswds-site#2646 for consistency. Let me know what you think!

```suggestion
    summaryAdditional: This allows the component to meet the "hoverable" standard outlined in [WCAG 1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).
```

Comment (amyleadem) [reply _data/changelogs/component-tooltip.yml]: I dig it. Thanks @mahoneycm. This was merged in [4770585](https://github.com/uswds/uswds-site/pull/2661/commits/47705851b17309e6de5d32661d9b7a0565e88199).

**Review by mejiaj — COMMENTED** [review]
@amyleadem this PR includes a lot of different changes. Are these captured anywhere else?

Tooltip note looks good.

**Comment by amyleadem** [comment]
@mejiaj Whoops! Thanks for flagging. The extra file changes came because I synced this PR with `main` instead of its base `release-3.8.1`. Should be better now.

**Pull request merged**